base/templates/base/base.html

```diff
@@ -1,129 +1,131 @@
+                            </p>
+                        </div>
+                        <div>
+                            {% block main-right-sidebar-content %}
+                            {% endblock main-right-sidebar-content %}
                         </div>
                     </div>
                 </div>
-
             </div>
+            {% endblock main-right-sidebar %}
 
-            <nav class="navbar navbar-default  navbar-fixed-bottom">
-                  <div class="container-fluid" style="width:80%; float:left;">
-                    <ul class="nav navbar-nav">
-                        <li class="inactive"><a href="/about/">About us</a></li>
-                        <li class="inactive"><a href="/faq/">FAQ</a></li>
-                        <li class="inactive"><a href="/contact/">Contact us</a></li>
-                        <li class="inactive"><a href="/legal/impressum/">Impressum</a></li>
-                        <li class="inactive"><a href="/legal/datasec/">Datenschutzerklärung</a></li>
-                        <li class="inactive"><a href="/legal/tou/">Terms of use</a></li>
-                        <li class="inactive"><a href="http://opendefinition.org/od/2.0/en/" target="_blank">Open Data {% bootstrap_icon "new-window" %}</a></li>
-                        <li class="inactive"><a href="https://okfn.org/" target="_blank">Open Knowledge  {% bootstrap_icon "new-window" %}</a></li>
-                    </ul>
-                  </div>
-            </nav>
         </div>
+    </div>
+{% endblock main %}
+
+{% block footer %}
+    <nav class="navbar navbar-default  navbar-fixed-bottom">
+            <div class="container-fluid" style="width:80%; float:left;">
+            <ul class="nav navbar-nav">
+                <li class="inactive"><a href="/about/">About us</a></li>
+                <li class="inactive"><a href="/faq/">FAQ</a></li>
+                <li class="inactive"><a href="/contact/">Contact us</a></li>
+                <li class="inactive"><a href="/legal/impressum/">Impressum</a></li>
+                <li class="inactive"><a href="/legal/datasec/">Datenschutzerklärung</a></li>
+                <li class="inactive"><a href="/legal/tou/">Terms of use</a></li>
+                <li class="inactive"><a href="http://opendefinition.org/od/2.0/en/" target="_blank">Open Data {% bootstrap_icon "new-window" %}</a></li>
+                <li class="inactive"><a href="https://okfn.org/" target="_blank">Open Knowledge  {% bootstrap_icon "new-window" %}</a></li>
+            </ul>
+            </div>
+    </nav>
+{% endblock footer %}
 
+{% block body-bottom-js %}
+    {% block before-body-bottom-js %}{% endblock before-body-bottom-js %}
+    {% bootstrap_javascript jquery=1%}
+    <script type="text/javascript" src="{% static '/js/jsi18n'%}"></script>
+    {% block after-body-bottom-js %}{% endblock after-body-bottom-js %}
+{% endblock body-bottom-js %}
```

Comment:
Javascript section at the bottom with subsections. Normally only block `after-body-bottom-js` should be overwritten.